examples/trl/requirements.txt

```diff
@@ -1,4 +1,4 @@
-trl == 0.7.6
+trl == 0.7.8
```

Comment:
@regisss @libinta @yeonsily I upgraded to trl 0.7.8 due to a conflict with trl 0.7.6 and diffusers 0.26.0+ for LoRA training.